exporter/tanzuobservabilityexporter/transformer.go

```diff
@@ -164,6 +184,11 @@ func attributesToTags(attributes ...pdata.AttributeMap) map[string]string {
 		att.Range(extractTag)
 	}
 
+	if value, isFound := tags["source"]; isFound {
```

Comment:
Feels like we should pull out a constant of `labelSource = "source"` to `exporter.go`, and use that instead of `"source"` everywhere.

Reply:
Done